cmd/landscaper-controller/app/app.go

```diff
@@ -71,6 +77,84 @@ func (o *options) run(ctx context.Context) error {
 		return fmt.Errorf("unable to setup execution controller: %w", err)
 	}
 
+	///// WEBHOOK /////
```

Comment:
the whole `run` function now looks long.
can you factor the webhook registration out to a new function `func (o *options) registerWebhook`